### ISSUE TITLE
Accessibility (?)

### ISSUE THREAD

**Issue by tassoman** [fballiano/p4a#23]
Did P4A was born with accessibility in mind? How much accessible default template should be? Do you think Ajax should be disabled to be totally accessible?

I'm measuring my application's accessibility using http://validatore.it but I got back a few errors (25), I feel I can work on it, maybe I'll post some pull requests in the future.

**Comment by fballiano** [comment]
it could not be formally accessible but p4a implements some different paradigms to make it so (do not repeat the "edit-delete" buttons on every row for example or real access key usage)
but please do not work on v3.x cause it's going to be completely rewritten :)

anyway thanks!

**Comment by tassoman** [comment]
Which version is stable release, and which is development?

**Comment by fballiano** [comment]
stable is 3.8 on master branch.

dev at the moment is not tagged but it's on the 4.0something branch

**Comment by tassoman** [comment]
We need accessibility in production servers so I'll go for hack the stable release core.
Maybe I'll fork p4a_psr_4_x branch for submissions.
Best regards and thank you very much for support.

**Comment by fballiano** [comment]
thanks to you, just now the 4.0 branch doesn't have anything about UI sorry.
I did a lot of work actually on bootstrap but now i'm again doubtful about the choice and I started thinking again to dojo :(

**Comment by tassoman** [comment]
As far I know, Bootstrap 3.x It's quite ok with accessibility adding wai-aria support.
I don't really know Dojo well to say about it.

Edit: after a short googling I've just found [DojoBootstrap](http://dojobootstrap.com/) :smiley:

**Comment by fballiano** [comment]
i know those kind of projects but they're not well maintained, a lot are based on bootstrap2 without plans about moving to 3, or same thing about dojo version.

**Comment by tassoman** [comment]
[Dijit](http://dojotoolkit.org/reference-guide/1.9/dijit/a11y/statement.html) utilizes the under development W3C Accessible Rich Internet Applications (ARIA) specification so that all dijit widgets are accessible using the latest versions of the JAWS or Window-Eyes screen readers with Firefox 3.

This is a [PayPal Accessibility plugin](https://github.com/paypal/bootstrap-accessibility-plugin) for Bootstrap3.

I agree with you is an hard choice.

**Comment by helpse** [comment]
Hello fballiano. I would recommend you bootstrap 3, since it's well mantained by a pro community. I can help on it.
